features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/crypto/identity/IdentityChangeStateView.kt

```diff
@@ -32,7 +33,8 @@ fun IdentityChangeStateView(
     // Pick the first identity change to PinViolation
     val pinViolationIdentityChange = state.roomMemberIdentityStateChanges.firstOrNull {
         // For now only render PinViolation
-        it.identityState == IdentityState.PinViolation
+        it.identityState == IdentityState.PinViolation ||
+            it.identityState == IdentityState.VerificationViolation
```

Comment:
Can you remove the 2 comments above please? Looks like there are not true anymore :)

Also probably `pinViolationIdentityChange` could be rename to something more generic like `firstRoomMemberIdentityStateChange`.

Reply:
done